pkgs/development/libraries/gstreamer/devtools/default.nix

```diff
@@ -54,6 +63,7 @@ stdenv.mkDerivation (finalAttrs: {
       stripLen = 2;
       hash = "sha256-CpBFTmdn+VO6ZeNe6NZR6ELvakZqQdaF3o3G5TSDuUU=";
     })
+    ./update-static-files.patch
```

Comment:
Could you also open a PR against GStreamer and link it in a comment, so that this is not a downstream patch forever?

Reply:
good point, opened https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/9208 and added a link to it